packages/core/src/services/documents/create.ts

```diff
@@ -25,7 +25,7 @@ export async function createNewDocument(
     content,
   }: {
     workspace: Workspace
-    user: User
+    user?: User
```

Comment:
Instead of making it optional. Pick the first user of the workspace and assign to it

Reply:
mmmm strange, but will do.